### ISSUE TITLE
error with raster-dev

### ISSUE THREAD

**Issue by rhijmans** [AndyPL22/ForestTools#22]
When checking ForestTools with the development version of "raster" I see:

```
forestTools
Quitting from lines 119-127 [unnamed-chunk-10] (treetop_analysis.Rmd)
Error: processing vignette 'treetop_analysis.Rmd' failed with diagnostics:
identicalCRS(x, y) is not TRUE
--- failed re-building 'treetop_analysis.Rmd'
```

This is due to changes in "raster" related to the changes in the "sp family" packages. I did not look into the details for this particular error, but comparing CRSs is always tricky, but now more so. sp::identicalCRS does not work well with Raster* objects anymore. You may want to "CRS(Raster*)" before comparing or use `terra::same.crs(x, y)`.

Can you please fix this so that things won't break on CRAN?

**Comment by andrew-plowright** [comment]
Thanks Robert. I have a version of ForestTools on Github (
https://github.com/andrew-plowright/ForestTools ) that has completely
substituted raster and sp with terra and sf. But it is failing a bunch of
R-CMD checks that I have not yet figured out, and so I haven't pushed the
new version onto CRAN yet. Soon, I hope!

Andrew

**Comment by andrew-plowright** [comment]
Resolved with version 1.0.0 of ForestTools
